### PR TITLE
Update docs/how-to-open-a-pull-request.md for grammar and readability

### PR DESCRIPTION
**Pull request by allthedoll**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

---

This pull request updates `docs/how-to-open-a-pull-request.md`:

- Fixes minor grammatical errors
- Rephrases portions for readability
- Fixes verb tense consistency issues
- Adds clarity to instructions
- Adds a link to keyword documentation

### CODE REVIEW

File: docs/how-to-open-a-pull-request.md [hunk 1]
```diff
@@ -37,13 +37,13 @@ An example is `fix(learn): Fixed tests for the do...while loop challenge`.
...
-      automatically close the existing issue, if the PR is accepted and merged.
+    - If the PR is meant to address an existing GitHub Issue then, at the end of
+      your PR's description body, use the keyword _Closes_ with the issue number to [automatically close that issue if the PR is accepted and merged](https://help.github.com/en/articles/closing-issues-using-keywords).
+      
+      > Example: `Closes #123` will close issue 123
 
 5. Indicate if you have tested on a local copy of the site or not.
 
-    This is very important when you are making changes that are not just making edits to text content such as a Guide article verbiage. Examples of changes needing local testing would include JavaScript, CSS, or HTML which could change the functionality or layout of a page.
+    This is very important when you are making changes that are not just making edits to text content, like documentation or a Guide article. Examples of changes needing local testing would include JavaScript, CSS, or HTML which could change the functionality or layout of a page.
```
Comment (scissorsneedfoodtoo) [inline docs/how-to-open-a-pull-request.md]: The Guide articles have been moved to the forum, so we can reword the first sentence to something like:

This is very important when making changes that are not just edits to text content like documentation or a challenge description. Examples of changes that need local testing include JavaScript, CSS, or HTML which could change the functionality or layout of a page.

**Comment by scissorsneedfoodtoo** [comment]
@allthedoll, thank you for taking the time to update our documentation. I made a couple of suggestions, so please take a look when you can and let us know what you think.

**Comment by moT01** [comment]
I have made the suggested changes, this should be good to go.

File: docs/how-to-open-a-pull-request.md [hunk 2]
```diff
@@ -37,13 +37,13 @@ An example is `fix(learn): Fixed tests for the do...while loop challenge`.
 
     - You will be presented with a pull request template. This is a checklist that you should have followed before opening the pull request.
 
-    - Fill in the details as they seem fit you. This information will be reviewed and decide whether or not, your pull request is going to be accepted.
+    - Fill in the details as you see fit. This information will be reviewed, and the reviewers will decide whether or not your pull request is accepted.
```
Comment (RandellDawson) [inline docs/how-to-open-a-pull-request.md]:
```suggestion
    - Fill in the details as you see fit. This information will be reviewed and reviewers will decide whether or not your pull request is accepted.
```

**Review by RandellDawson — CHANGES_REQUESTED** [review]
See my suggested changes.

File: docs/how-to-open-a-pull-request.md (hunk 2 above)
Comment (moT01) [reply docs/how-to-open-a-pull-request.md]: This is not my PR, and I cannot commit this suggestion via the interface. I made the original suggested changes on behalf of the OP in hopes to get this PR merged and I think it's fine how it is. If you want it this way, you should make the changes.

Comment (RandellDawson) [reply docs/how-to-open-a-pull-request.md]: I am going to add a commit.

**Review by RandellDawson — APPROVED** [review]

**Review by moT01 — APPROVED** [review]
LGTM @allthedoll 🎉 Congratulations on your first contribution to freeCodeCamp 🎉

**Pull request merged**